Header header
geometry_msgs/Point goal_point
float64 reward
